runestone/activecode/js/MD5.js

```diff
@@ -0,0 +1,203 @@
+/**
```

Comment:
How would you like me to publicly link an outside source to this repository. I am not quite sure how to do that.